build-tools/automation/azure-pipelines.yaml

```diff
@@ -14,7 +14,7 @@ resources:
   - repository: yaml
     type: github
     name: xamarin/yaml-templates
-    ref: refs/heads/main
+    ref: refs/heads/dev/pjc/sign-use-nuget
```

Comment:
I assume this means that there is some "other" PR in xamarin/yaml-templates -- perhaps xamarin/yaml-templates#160? -- that should be merged *first*?

Reply:
That's correct, I should marked this as a draft after hitting the issue fixed by xamarin/yaml-templates#160. That PR has now been merged and the branch reference has been updated here.